src/main/java/com/openlattice/launchpad/Launchpad.java

```diff
@@ -59,7 +60,12 @@ public static void main( String[] args ) throws ParseException, IOException {
             System.exit( -1 );
         }
 
-        IntegrationRunner.runIntegrations( config );
+        try ( SparkSession session = IntegrationRunner.Companion.configureOrGetSparkSession( config )) {
+            Runtime.getRuntime().addShutdownHook( new SparkSessionShutdownHook( session ) );
```

Comment:
I don't think we need both of these? Sesssion.close() will be called from the try-with-resources statement, and does the same thing.

Reply:
Yeah I was wondering about that. Is there a difference in behavior in the case that someone manually kills the process? ie Shutdown hook running but auto-closing not running?